src/components/app/app.js

```diff
@@ -3,9 +3,14 @@ import template from "./app.html";
 import { DrawerComponent } from "../drawer/drawer";
 import { ToolbarComponent } from "../toolbar/toolbar";
 import { TransactionsComponent } from "../transactions/transactions";
-import { AddAccountComponent } from "../add-account-dialog/add-account-dialog";
+import { AddAccountComponent } from "../accounts/add-account-dialog/add-account-dialog";
 import { AddTagComponent } from "../add-tag-dialog/add-tag-dialog";
 import { AboutComponent } from "../about-dialog/about-dialog";
+import { PieChartComponent } from "../pie-chart/pie-chart";
```

Comment:
you have charts and table in app again. please, remove them. they are in transaction component